### ISSUE TITLE
Real Feel Temp 

### ISSUE THREAD

**Issue by Fatcakes** [martykan/forecastie#321]
Any way we can get that info on the app?

**Comment by robinpaulson** [comment]
As in windchill (https://en.wikipedia.org/wiki/Wind_chill) factor? if so, there's a request for that already: #301

**Comment by robinpaulson** [comment]
Ah: https://en.wikipedia.org/wiki/Feels_like_temperature ?

**Comment by Fatcakes** [comment]
Yeah, it's windchill in the winter, and heat index in the summer.

**Comment by robinpaulson** [comment]
As I suggested in #301 , we can calculate this even if OWM doesn't provide it. Either way, code and a PR is the quickest way to get this feature added. @Fatcakes are you willing to do this?

**Comment by robinpaulson** [comment]
@Fatcakes Would you be willing to look up the info for getting the data from OWM? I may take a look at it.

**Comment by robinpaulson** [comment]
I'm going to close this, as it's been superseded by #397. This comment will ensure the two issues are linked, meaning we keep the discussion and other information referenced.

**Comment by FridoDeluxe** [comment]
Weeks ago I've done some reasearch on this topic and it seems like calculating a meaningful real-feel temperature is a bit more complicated than just using wind chill and heat index. Still these two values are a good start.

Fortunately OWM seems to provide now a real-feel temperature: The JSON responses now contain an additional field named `feels_like`: 

> This temperature parameter accounts for the human perception of weather. ([source](https://openweathermap.org/current#current_JSON))

It should be relatively easy to include it into Forecastie - maybe after #397 to avoid overloading the UI.
